interface/wx/artprov.h

```diff
@@ -212,6 +214,7 @@ const char* wxART_WX_LOGO;
      @li @c wxART_CDROM
      @li @c wxART_REMOVABLE
      @li @c wxART_WX_LOGO (since 3.1.6)
+     @li @c wxART_PREV_SCREEN (since 3.1.6)
```

Comment:
```suggestion
     @li @c wxART_PREV_SCREEN (since 3.3.2)
```

Reply:
Also, it would be nice to explain how is this one different from `wxART_GO_BACK`.